src/Makefile.am

```diff
@@ -181,6 +181,7 @@ libbitcoin_server_a_SOURCES = \
   pow.cpp \
   rest.cpp \
   rpc/blockchain.cpp \
+  rpc/client.cpp \
```

Comment:
client.cpp is part of `libbitcoin_cli`, which is "cli: shared between bitcoin-cli and bitcoin-qt" if you need access to that then link that library. Don't include the compilation unit in two libraries.